common/lib/xmodule/xmodule/css/problem/edit.scss

```diff
@@ -23,39 +23,22 @@
       }
     }
 
-    .cheatsheet-toggle {
-      width: 21px;
-      height: 21px;
-      padding: 0;
-      margin: -1px 5px 0 15px;
-      border-radius: 22px;
-      border: 1px solid #a5aaaf;
-      background: #e5ecf3;
-      font-size: 13px;
-      font-weight: 700;
-      color: #565d64;
-      text-align: center;
-    }
   }
 }
 
 .simple-editor-cheatsheet {
   position: absolute;
-  top: 0;
-  left: 100%;
+  top: 41px;
```

Comment:
this is unrelated to this line but for Change #4 (only entry for this, I would update .editor-modes to be width 49% instead of 48% to minimize the amount of spacing the Editor and Settings buttons have to their right now that the window is a larger overall percentage width.